roles/edge-image-builder/templates/eib-config-mgmt.yaml.j2

```diff
@@ -0,0 +1,26 @@
+apiVersion: 1.0
+image:
+  imageType: raw
+  arch: x86_64
+  baseImage: {{ eib_source_image_name }}
+  outputImageName: {{ eib_mgmt_output_image_name }}
+operatingSystem:
+  rawConfiguration:
+    diskSize: {{ eib_image_size }}
+  kernelArgs:
+  - net.ifnames=1
+  users:
+  - username: root
+    encryptedPassword: "$6$Hq6XnrxLLviHzrkE$Qft13mh082uQZl6Wm09a7pUmf7ULt6vlr9u7koAV60v/LNs1YkseTSuBmr1SJ2yg.Ib2WY06YGldJbDGCC4ry1"
```

Comment:
A comment here on what is the password won't hurt I guess

Reply:
I removed the root user as we can log in via vm_user and that's what's already documented